apps/web-staking/src/services/web3.service.ts

```diff
@@ -263,77 +263,44 @@ export type OrderedRedemptions = {
 	closed: RedemptionRequest[];
 };
 
-export const getRedemptions = async (network: NetworkKey, walletAddress: string): Promise<OrderedRedemptions> => {
-	const storageKey = "redemptionRequests" + network + walletAddress;
-	const storage = localStorage.getItem(storageKey);
-	const cachedRedemptions = JSON.parse(storage || "[]");
-
+export const getRedemptions = async (network: NetworkKey, walletAddress: string, qty: number, offset: number): Promise<OrderedRedemptions> => {
 	const web3Instance = getWeb3Instance(network);
 	const esXaiContract = new web3Instance.web3.eth.Contract(esXaiAbi, web3Instance.esXaiAddress);
-	const numRedemptions = Number(await esXaiContract.methods.getRedemptionRequestCount(walletAddress).call());
 
-	const numCachedRedemption = cachedRedemptions.length;
+	const result = await esXaiContract.methods.getRedemptionsByUser(walletAddress, qty, offset).call();
+	const redemptionsFromChain = result[0];
+	const redemptionsCount = Number(result[1]);
```

Comment:
I tried de-structuring these like this `const [redemptions, totalRedemptions] = await esXaiContract.methods`  , but I could not get it to work.

Received console error: `TypeError: (intermediate value) is not iterable
    at getRedemptions`